src/api/api.ts

```
@@ -1,3 +1,4 @@
+import toast from "react-hot-toast";
```

Biome (2.1.2)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Unsafe fix: Remove the unused imports.


(lint/correctness/noUnusedImports)